src/Currencies.java

```diff
@@ -0,0 +1,22 @@
+import java.util.*;
+public class Currencies {
+    Map<String, Map<String,Double>> rates;
+    Currencies(){
+        rates= new HashMap<>();
+    }
+    public void addCurrency (String currency){
```

Comment:
Do you actually need that function? At the end adding currency alone wouldn't do a thing. You still have to provide rates, so maybe just remove this and add currency in `addRate` method

Reply:
bump